cmake/macros/macro_enable_if_supported.cmake

```diff
@@ -28,7 +28,7 @@ MACRO(ENABLE_IF_SUPPORTED _variable _flag)
   # actual errors with the -Werror switch:
   #
   IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    SET(_werror_string "-Werror ")
+    SET(_werror_string "")
   ELSE()
     SET(_werror_string "")
```

Comment:
You don't need the if-else if you set `_werror_string` to the same value regardless :-)

Reply:
I was only trying out something ;) #11305 seems to work for now. So I am closing this PR.